src/anniversary/sf33rd/Source/Game/FOLLOW02.c

```diff
@@ -0,0 +1,96 @@
+#include "common.h"
+#include "sf33rd/Source/Game/Com_Sub.h"
+#include "sf33rd/Source/Game/workuser.h"
+#include "structs.h"
+
+void Follow02(PLW *wk);
+void Follow02_0000(PLW *wk);
+void Follow02_0001(PLW *wk);
+void Follow02_0002(PLW *wk);
+void Follow02_0003(PLW *wk);
+
+typedef void (*FOLLOW02_Func)(PLW *wk);
+
+const FOLLOW02_Func Follow02_Tbl[4] = {
+    Follow02_0000,
+    Follow02_0001,
+    Follow02_0002,
+    Follow02_0003,
+};
+
+void Follow02(PLW *wk) {
+    Follow02_Tbl[(s16)Pattern_Index[(s32)wk->wu.id]](wk);
+}
+
+void Follow02_0000(PLW *wk) {
+    switch (*CP_Index[wk->wu.id]) {
```

Comment:
That should probably look like this:
```suggestion
    switch (CP_Index[wk->wu.id][0]) {
```

Same goes for other instances of this code